lizmap/modules/lizmap/controllers/geobookmark.classic.php

```diff
@@ -11,6 +11,8 @@
  */
 class geobookmarkCtrl extends jController
 {
+    protected $whiteParams;
```

Comment:
Here **PhpStan** fllagged the error `Access to an undefined property geobookmarkCtrl::$whiteParams.` so i think we need to define `protected $$whiteParams`